common/decrees/00_decree.txt

```diff
@@ -297,21 +297,17 @@ decree_worker_exploitation = {
 		}
 	}
 	modifier = {
-		state_migration_pull_unincorporated_mult = -0.6
-		state_migration_pull_mult = -0.5
 		building_throughput_add = 0.2
-		state_unincorporated_starting_wages_mult = -0.12
-		building_subsistence_output_mult = -0.25
-		state_laborers_mortality_mult = 0.01
-		state_machinists_mortality_mult = 0.01
-		state_farmers_mortality_mult = 0.01
-		state_slaves_mortality_mult = 0.015
-		state_pop_qualifications_mult = -0.1
-		state_radicals_from_sol_change_mult = 0.05
-		building_minimum_wage_mult = -0.35
-		building_working_conditions_mult = 0.15
-	}
-	cost = 50 # was 40
+		state_welfare_payments_add = -0.5
+		state_laborers_mortality_mult = 0.04
+		state_machinists_mortality_mult = 0.035
+		state_farmers_mortality_mult = 0.025
+		state_engineers_mortality_mult = 0.025
+		state_peasants_mortality_mult = 0.05
+		state_slaves_mortality_mult = 0.05
+		state_radicals_from_sol_change_mult = 0.3333
+	}
+	cost = 55 # was 40
```

Comment:
At some point you should figure out whether the building throughput doesn't add enough wealth such that it negates the mortality. But that will be late game anyways....